pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -458,6 +458,35 @@ type EndpointIdentity struct {
 	LabelsSHA256 string `json:"labelsSHA256,omitempty"`
 }
 
+// +genclient
```

Comment:
Add:
```
// +genclient:nonNamespaced
```

To not require to specify the namespace in client code

Reply:
done! excellent